src/Sarif/RuntimeConditions.cs

```diff
@@ -11,51 +11,98 @@ namespace Microsoft.CodeAnalysis.Sarif
     // code paths. These conditions are a combination of fatal
     // and non-fatal circumstances
     [Flags]
-    public enum RuntimeConditions : uint
+    public enum RuntimeConditions : long
```

Comment:
>long [](http://example.com/codeflow?start=36&length=4)

@michaelcfanning we changed this to long, should we change the usage of it to use long as well, 
e.g. I see the run result was int, should we also change to long?
``` CSharp
abstract public int Run(T options);
```